views/tasks.handlebars

```diff
@@ -6,19 +6,45 @@
 
     </div>
     {{#each userTasks}}
-    <div class="col-md-3 col-sm-12 mb-3">
-      <div class="card">
-        <div class="card-body">
-          <h5 class="card-title">{{Task.title}}</h5>
-          <div class="card-text">{{Task.text}}</div>
-          <div class="text-center">
-            <button class="btn btn-primary mt-4">Complete</button>
+    <div class="col-md-9 col-sm-12">
+      <div class="row">
+        <div class="col-md-4 col-sm-12 mb-4">
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title">{{Task.title}}</h5>
+              <div class="card-text">{{Task.text}}</div>
+              <div class="text-center">
+                <button class="btn btn-danger mt-4">Complete</button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+     {{/each}}
+
+{{!-- flex-row-reverse will hold progress bar on the rightmost column when above handlebars populates cards --}}
+    <div class="col-md-3 col-sm-12 flex-row-reverse">
```

Comment:
I tried this locally and it wasn't looking right. The progress bar was just attaching to the end of the cards and not really displaying in the upper right hand corner. I'll push a change to this branch with the changes I think need to be made.